easybuild/easyconfigs/m/matplotlib/matplotlib-2.0.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '2.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('freetype', '2.7'),
+    ('libpng', '1.6.26'),
```

Comment:
@boegel do you mean the `Intel` version? I wanted to keep compatibility with what's available in `foss/2016b`, which is currently 2.7 and 1.6.26, respectively.